guru/gmt-eco-system.sh

```diff
@@ -20,16 +20,16 @@ gmt begin gmt-eco-system png
 	# Place logo without label
 	gmt logo -DjCT+w1.5i+o0/0.05i -Sn
 	# Add label and purpose
-	echo " 0 0.25 THE GMT ECOSYSTEM" | gmt text -F+f12p,Helvetica-Bold,92/102/132
-	echo " 0 0.10 generic-mapping-tools.org" | gmt text -F+f6p,Helvetica,92/102/132
-	echo " 0 -0.01 Data Processing \035 Plots \035 Animations" | gmt text -F+f8.5p,Times-Italic,92/102/132
+	echo " 0 0.29 generic-mapping-tools.org" | gmt text -F+f6p,Helvetica-Bold,92/102/132
+	echo " 0 0.14 Data Processing \035 Plots \035 Animations" | gmt text -F+f8.5p,Times-Italic,92/102/132
+	echo " 0 0 Multi-platform \035  Multi-language" | gmt text -F+f8.5p,Times-Italic,92/102/132
```

Comment:
Makes the capitalization consistent
```suggestion
	echo " 0 0 Multi-Platform \035  Multi-Language" | gmt text -F+f8.5p,Times-Italic,92/102/132
```